lib/datasets/ita-corpus.rb

```diff
@@ -0,0 +1,72 @@
+require 'csv'
+
+require_relative 'dataset'
+
+module Datasets
+  class ITAcorpus < Dataset
+    Record = Struct.new(:id,
+                        :sentence)
+
+    def initialize
+      super
+      @metadata.id = 'ita-corpus'
+      @metadata.name = 'ITA-corpus'
+      @metadata.url = 'https://github.com/mmorise/ita-corpus'
+      @metadata.licenses = ['Unlicense']
+      @metadata.description = lambda do
+        fetch_readme
+      end
+    end
+
+    def each
+      return to_enum(__method__) unless block_given?
+
+      open_emotion_data do |text|
+        text.each do |row|
```

Comment:
`#each` is an alias of `#each_line` but using `#each_line` is better here. Because it's more self descriptive.

```suggestion
        text.each_line  do |line|
```

FYI: We can use `chomp: true` option here:

```suggestion
        text.each_line(chomp: true) do |line|
```

Reply:
I'll remember this. Thank you.